scripts/frame/main.js

```diff
@@ -1,11 +1,24 @@
 // Sub-implementation of lychee -------------------------------------------------------------- //
 
-let lychee = {
+// TODO: Find out and explain: Here we declare a global (empty) object `lychee`; we also declare one in `./main/lychee.js`. Why don't they interfere with each other? How do we end up with **one** `lychee` object which contains the properties and methods of both objects?!
```

Comment:
This JS is loaded only when you access `/frame` instead of `/gallery` as a result you do not need all the other info that are provided in `main/lychee.js`